### ISSUE TITLE
Xcode workspace integrity issue related to `langs`

### ISSUE THREAD

**Issue by aburgel** [malcommac/SwiftDate#791]
I recently noticed the following error in xcode. It doesn't seem to cause any breakage, my app builds, runs and tests without issue.

<img width="237" alt="Screen Shot 2022-01-18 at 3 14 44 PM" src="https://user-images.githubusercontent.com/341478/150011782-457f8b10-bab7-4438-956f-38271d89af16.png">

I think it's related to SwiftDate because I also see this error in my fastlane builds.

```
2022-01-18 18:33:53.756 xcodebuild[25156:62068] [MT] IDEFileReferenceDebug: [Load] <IDEFileReference, 0x7fd8b5a0bda0: name:langs path:absolute:/Users/runner/Library/Developer/Xcode/DerivedData/xxx/SourcePackages/checkouts/SwiftDate/Sources/SwiftDate/Formatters/RelativeFormatter/langs> Failed to load container at path: /Users/runner/Library/Developer/Xcode/DerivedData/xxx/SourcePackages/checkouts/SwiftDate/Sources/SwiftDate/Formatters/RelativeFormatter/langs, Error: Error Domain=com.apple.dt.IDEContainerErrorDomain Code=6 "Cannot open "langs" as a "Folder" because it is already open as a "Swift Package Folder"." UserInfo={NSLocalizedDescription=Cannot open "langs" as a "Folder" because it is already open as a "Swift Package Folder".}
```

`langs` [is referenced in Package.swift](https://github.com/malcommac/SwiftDate/blob/master/Package.swift#L21), but I'm not sure whether that's related or why it's causing this issue.

I'm using Xcode 13.2.1.

**Comment by andriyslyusar** [comment]
I have the same issue using Xcode 13.2.1 and the same fastlane output.

**Comment by ksandiego** [comment]
I'm currently facing the same issue as well. It seems to only appear during tests; however, the app builds and tests fine.

**Comment by GeorgeSealy** [comment]
Also getting the same issues, and it's causing CI builds to fail. Is anyone else seeing / monitoring this issue? https://forums.swift.org/t/xcode-and-swift-package-manager/44704/5 mentions possible work arounds, but they don't seem to work in this case.

**Comment by malcommac** [comment]
fixed in 7.0.0
